docs/UserGuide.md

```diff
@@ -31,104 +31,249 @@
+> Delete a task/event.<br>
+> The index refers to the index number shown in the most recent listing.<br>
+> The index **must be a positive integer** 1, 2, 3, ...<br>
 
-Examples: 
+Examples:
 * `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
   `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  Delete the first task in the list.
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+#### Mark a task as complete: `complete`
+Marks a task as completed.<br>
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+Format: `complete INDEX`
 
-Examples: 
+> When you are done with the task, you can mark it as complete instead
+> of deleting it.<br>
+>> Marking a task as completed will also hide it from the current listing results.<br>
+>
+> The index refers to the index number shown in the most recent listing.<br>
+> The index **must be a positive integer** 1, 2, 3, ...<br>
+
+> <img src="images/Ui-CompleteCommand.png" width="600">
+
+Example:
 * `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+  `complete 1`<br>
+  Mark the first task in the list as complete.
 
-#### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
-Format: `clear`  
+#### Undo the last action: `undo`
+To undo any last action that modifies the task database (e.g. deleting task).<br>
+Format: `undo [last STEPS]`
 
-#### Exiting the program : `exit`
-Exits the program.<br>
-Format: `exit`  
+> You can undo multiple steps by specifying the number of steps to undo.
 
-#### Saving the data 
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
-There is no need to save manually.
+Example:
+* `undo`<br>
+  Undo the most recent action.
+* `undo last 5`<br>
+  Undo the last 5 actions.
+
+#### Clearing all entries: `clear`
+Removes all tasks from the data storage file.<br>
+
+Format: `clear`
+
+> Since this is a potentially destructive action, a confirmation would be shown 
+> before the tasks are removed.
+
+#### Relocate the data storage location: `relocate`
+Designates a new data storage location.<br>
+
+Format: `relocate PATH`
+
+#### Autocomplete and suggestions
+Suggested command keywords, dates, sorting order, and tags are shown as you type.
+Use the `tab` key to autocomplete using the suggestion.
+
+#### Saving the Data
+Changes are automatically saved after each command that changes the data.
+No manual saving required.
+
+#### Multiple Storage Files
+Multiple storage files are allowed for you to store different schedules 
```

Comment:
can you think of providing this as a command instead? 
imagine a user like me having 2 lists, "research" and "TA".
i can start the program, load what list was there on close at start up (say "research"), then if i need to change the list, i don't need to close and open. say something like `changelist` and provide the list name. that would be cool :-D